static/app/components/core/slider/index.chonk.tsx

```diff
@@ -33,7 +33,7 @@ export function Slider({formatLabel = passthrough, ref, ...props}: SliderProps)
           props.onChange?.(newValue, e);
         }}
       />
-      {props.disabled ? null : (
+      {props.disabled || !formatLabel(value) ? null : (
```

Comment:
we now call `formatLabel(value)` twice. Small thing but we could easily extract it to:

```
const formattedLabel = formatLabel(value)
```

then check for it and render it in the `SliderLabel`